themes/default/content/resources/intro-to-infrastructure-as-code-with-yaml/index.md

```diff
@@ -0,0 +1,59 @@
+---
+preview_image: ""
+hero:
+  image: /icons/containers.svg
+  title: Intro to Infrastructure as Code with YAML
+main:
+  duration: 90 minutes
+  presenters:
+    - name: Laura Santamaria
+      role: Developer Advocate, Pulumi
+  learn:
+    - How to use YAML with Pulumi.
+    - How to provision Docker containers with Pulumi.
+  title: Intro to Infrastructure as Code with YAML
+  description: >-
+    In this workshop, we’re going to learn more about cloud computing
+    and Infrastructure as Code by exploring how to use Pulumi to build,
+    configure, and deploy a real-life, modern application using Docker. We will
+    create a frontend, a backend, and a database to deploy the Pulumipus Boba
+    Tea Shop, and along the way, learn more about how Pulumi works to make
+    managing all of these different moving pieces a little bit less painful!
+
+
+    If you want to code along, you’ll need a [free Pulumi SaaS account](https://app.pulumi.com/signup/) and [the Pulumi CLI](https://www.pulumi.com/docs/get-started/install/).
+  sortable_date: 2022-12-15T18:00:00.000Z
+event_data:
+  name: Intro to Infrastructure as Code with YAML
+  start_date: 2022-12-10T18:00:00.000Z
+  end_date: 2022-12-10T19:30:00.000Z
+  description: >-
+    In this workshop, we’re going to learn more about cloud computing
+    and Infrastructure as Code by exploring how to use Pulumi to build,
+    configure, and deploy a real-life, modern application using Docker. We will
+    create a frontend, a backend, and a database to deploy the Pulumipus Boba
+    Tea Shop, and along the way, learn more about how Pulumi works to make
+    managing all of these different moving pieces a little bit less painful!
+
+
+    If you want to code along, you’ll need a free Pulumi SaaS account and the Pulumi CLI.
+  url: https://www.pulumi.com/resources/intro-to-infrastructure-as-code-with-yaml
+form:
+  hubspot_form_id: 9812f461-677d-443c-bf3b-76d3d7c27310
+  salesforce_campaign_id: 701Du0000009P9GIAU
+  gotowebinar_key: ""
+featured: false
+pre_recorded: false
+pulumi_tv: false
+unlisted: false
+gated: true
+type: webinars
+external: false
+block_external_search_index: false
+aws_only: false
+title: Intro to Infrastructure as Code with YAML
+meta_desc: Learn the fundamentals of Infrastructure as Code (IaC) with Pulumi,
+  using Docker, and YAML. Build and deploy a full stack app. Source code
+  available.
+url_slug: intro-to-infrastructure-as-code-with-yaml
```

Comment:
I had to change the url from "introduction-to-infrastructure-as-code-with-yaml" to "intro-to-infrastructure-as-code-with-yaml" since the other one was aready being used by a different workshop.